# Automatically Generated in 2017-06-12 22:33:47.452922
# MESSAGE: ODOMETER_DELTA
# Description:Measurement of Odometry Sensor within a time-period
uint8 ID = 138
uint8 sysid
uint8 compid
uint32 timestamp
float32 delta
float32 xdist
float32 ydist
float32 zdist
float32 xvelocity
float32 yvelocity
float32 zvelocity
int16 quality
